src/helpers/style-dictionary/formats/tokens.js

```diff
@@ -38,15 +51,79 @@ const renderOutput = (isJS, tokenFamily, tokensArray, customValueOrType) => {
 
   return (
     declaration(isJS) +
-    `${tokenFamily} : {` +
+    `${tokenFamily}: {` +
     tokensArray.map((token) => {
       return `${token.name} : ` + renderValueOrType(token);
     }) +
-    `}${commaOrColon(isJS)}`
+    `}${commaOrColon(isJS)}\n`
   );
 };
 
+const renderTypographyOutput = (isJS, tokensArray, customValueOrType) => {
+  const renderValueOrType = (token, isFontWeight) =>
+    customValueOrType(token, isJS, isFontWeight);
+
+  /* Output the typography sub values
+   *
+   * ex: {
+   *    fontFamily: "Mulish"
+   *    fontSize: "24px"
+   * }
+   */
+  const getTypographySubValues = (typographyValues) => {
+    return Object.entries(typographyValues).map((key, value) => {
+      const propertyName = key[0];
+      const propertyValue = key[1];
+
+      return `${propertyName}: ${renderValueOrType(
+        propertyValue,
+        propertyName === `fontWeight`
+      )} `;
+    });
+  };
+
+  /* Pluck out the main Typography types, into two sizes
+   *
+   * 1. Root typography types
+   * ex: h1, h2, h3, etc.
+   *
+   * 2. Mobile typography types
+   * ex: mobileH1, mobileH2
+   */
```

Comment:
ty for the extra comments, we were in desperate need lol 🙏

Reply:
It was my own rubber duck to try and help me understand 😅